community/FfDL-H2Oai/h2o-model/manifest-h2o.yml

```diff
@@ -0,0 +1,28 @@
+name: h2o3_automl
+description: H2O-3 Model Using H2O AutoML
+version: "1.0"
+learners: 2
+gpus: 0
+cpus: 0.5
+memory: 1Gb

```

Comment:
After running few tests with this example, it's better to run with 1 cpu and 2Gb of memory.